Servers/Administration/OJS.Servers.Administration/Properties/launchSettings.json

```diff
@@ -38,7 +38,8 @@
         "DISTRIBUTOR_API_KEY": "f04101a2-0ae3-46a2-8f4d-057de2fd4e69",
         "LOGGER_FILES_FOLDER_PATH": "../../../Logs",
         "SHARED_AUTH_COOKIE_DOMAIN": "localhost",
-        "REDIS_CONNECTION_STRING": "redis,password=redisPass123,ConnectTimeout=10000,abortConnect=false"
+        "REDIS_CONNECTION_STRING": "redis,password=redisPass123,ConnectTimeout=10000,abortConnect=false",
+        "UI_URL": "http://localhost:5002"
```

Comment:
There are other url configurations above, put this above/under them so the configuration is easier to read

Reply:
Done.